packages/host/tests/unit/services/ai-assistant-panel-service-test.ts

```diff
@@ -0,0 +1,124 @@
+import { setupTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+import AiAssistantPanelService from '@cardstack/host/services/ai-assistant-panel-service';
+import type MonacoService from '@cardstack/host/services/monaco-service';
+
+module('Unit | Service | ai-assistant-panel-service', function (hooks) {
+  setupTest(hooks);
+
+  test('focusPillCodeRange returns undefined when there is no selection', function (assert) {
+    let service = this.owner.lookup(
+      'service:ai-assistant-panel-service',
+    ) as AiAssistantPanelService;
+
+    let monacoService = this.owner.lookup(
+      'service:monaco-service',
+    ) as MonacoService;
```

Comment:
I think if you use `getService` as seen in other test files you get the type automatically